DetDescr/src/DetDescr/EcalGeometry.cxx

```diff
@@ -619,26 +588,20 @@ double EcalGeometry::distanceToEdge(EcalID id) const {
 }
 
 bool EcalGeometry::isInside(double normX, double normY) const {
-  if (verbose_ > 2)
-    std::cout << TString::Format(
-                     "[isInside] Checking if normXY=(%.2f,%.2f) is inside.",
-                     normX, normY)
-              << std::endl;
+  ldmx_log(trace) << TString::Format(
+      "[isInside] Checking if normXY=(%.2f,%.2f) is inside.", normX, normY);
```

Comment:
I would request doing the `std::setprecision(2)` rather than `TString::Format`. This current way constructs the `TString` even if the log is thrown away while the stream (`<<`) based solution avoids constructing the message if we're ignoring the log.